configure.ac

```diff
@@ -333,4 +334,17 @@ AM_CONDITIONAL([ENABLE_SHARED], [test $enable_shared = yes])
 # The Makefile needs to know if we are cross-compiling.
 AM_CONDITIONAL([CROSS_COMPILING], [test $cross_compiling = yes])
 
+XCRYPT_VERSION_MAJOR=$(
+    echo $PACKAGE_VERSION | $AWK '{split($0,a,"."); print a@<:@1@:>@}'
```

Comment:
This can be done more simply: `| $AWK -F. '{ print $1 }'` or `| cut -d. -f1`.  And then you don't need to use `@<:@`.

Reply:
Fixed.